index.html

```diff
@@ -8,6 +8,36 @@
   </head>
   <body class="page">
     <div class="page__container">
+      <section class="advantages"> 
+        <div class="advantages__preview"><p class="advantages__heading">Устали от серости мегаполиса?</p><p class="advantages__heading-text">Нам есть, что вам<br> предложить!</p></div>
+        <div class="advantages__container">
+          <div class="advantages__pictures">
```

Comment:
@NazarPauk There is also an opportunity to make our BEM naming more readable by specifying our image wrapper as an `advantages__picture-wrapper`

```html
<div class="advantages__pictures">
  <div class="advantages__picture-wrapper">
    <img class="advantages__picture advantages__picture_hidden" src="images/advantages-iphone-grey-desktop.png" alt="Телефон с серым изображением мусорников">
  </div>
  <!-- other images -->
</div>
```

Reply:
Done✅